LongWarOfTheChosen/Config/XComLW_Overhaul.ini

```diff
@@ -597,15 +597,15 @@ INSTANT_BUILD_TIMES=TRUE
 ;Mag
 +ItemTable=(ItemTemplateName="Bullpup_MG",	Slots=3,	Starting=false,	Infinite=false,	Buildable=true,	RequiredTech1="MagnetizedWeapons",		RequiredTech2="",	SupplyCost=40,	AlloyCost=3,	CrystalCost=3,	CoreCost=0,	SpecialItemTemplateName="",	SpecialItemCost=0,	TradingPostValue=20, RequiredEngineeringScore=10, Tier = 150, PointsToComplete=100, InventoryImage="img:///UILibrary_XPACK_Common.MagSMG_Base")
 +ItemTable=(ItemTemplateName="VektorRifle_MG",	Slots=3,	Starting=false,	Infinite=false,	Buildable=true,	RequiredTech1="MagnetizedWeapons",		RequiredTech2="",	SupplyCost=50,	AlloyCost=3,	CrystalCost=3,	CoreCost=0,	SpecialItemTemplateName="",	SpecialItemCost=0,	TradingPostValue=25, RequiredEngineeringScore=10, Tier = 150, PointsToComplete=100, InventoryImage="img:///UILibrary_XPACK_Common.MagVektor_Base")
-+ItemTable=(ItemTemplateName="ShardGauntlet_MG",	Slots=0,	Starting=false,	Infinite=false,	Buildable=true,	RequiredTech1="MagnetizedWeapons",		RequiredTech2="",	SupplyCost=35,	AlloyCost=3,	CrystalCost=3,	CoreCost=0,	SpecialItemTemplateName="CorpseAdventStunLancer",	SpecialItemCost=1,	TradingPostValue=15, RequiredEngineeringScore=10, Tier = 150, PointsToComplete=100, InventoryImage="img:///UILibrary_XPACK_StrategyImages.Inv_MagTGauntlet")
-+ItemTable=(ItemTemplateName="WristBlade_MG",	Slots=0,	Starting=false,	Infinite=false,	Buildable=true,	RequiredTech1="MagnetizedWeapons",		RequiredTech2="",	SupplyCost=30,	AlloyCost=3,	CrystalCost=3,	CoreCost=0,	SpecialItemTemplateName="CorpseAdventStunLancer",	SpecialItemCost=1,	TradingPostValue=15, RequiredEngineeringScore=10, Tier = 150, PointsToComplete=100, InventoryImage="img:///UILibrary_XPACK_StrategyImages.Inv_MagSGauntlet")
++ItemTable=(ItemTemplateName="ShardGauntlet_MG",	Slots=0,	Starting=false,	Infinite=false,	Buildable=true,	RequiredTech1="PlatedArmor",		RequiredTech2="AutopsyAdventStunLancer",	SupplyCost=35,	AlloyCost=3,	CrystalCost=3,	CoreCost=0,	SpecialItemTemplateName="CorpseAdventStunLancer",	SpecialItemCost=1,	TradingPostValue=15, RequiredEngineeringScore=10, Tier = 150, PointsToComplete=100, InventoryImage="img:///UILibrary_XPACK_StrategyImages.Inv_MagTGauntlet")
```

Comment:
Why tie the Shard Gauntlet to armour tech? It's a bit confusing considering none of the other weapon techs have such a requirement.

Reply:
Removed armor tech requirement.